### DISCUSSION TITLE
Would it make sense to minify the embed privacy javascript?

### DISCUSSION THREAD

**Discussion by oldrup** [epiphyt/embed-privacy#218] [Ideas]
A few web page performance checks have suggested that I minify the embed privacy javascript.

Some plugins serves a minified js under normal conditions, and a non-minified when WP_DEBUG is enabled.

It appears that ~50% reduction can be achieved when minified (screenshot below)

I can configure my caching plugin to minify, but I'd rather not, as its very much "all or nothing".

Thoughts?

![image](https://github.com/epiphyt/embed-privacy/assets/17141205/f1030a32-74d9-4cee-bd96-7078916d9965)

**Comment by MatzeKitt** [comment]
This should already be the case.

It’s part of the webpack configuration: https://github.com/epiphyt/embed-privacy/blob/main/webpack.config.js#L12

And also the minified version is registered and later printed if `WP_DEBUG` is `false`: https://github.com/epiphyt/embed-privacy/blob/main/inc/class-embed-privacy.php#L1686-L1689

**Comment by oldrup** [comment]
🤦‍♂️ Uhm. Nothing to see here, move along 😆
![image](https://github.com/epiphyt/embed-privacy/assets/17141205/3880dd24-2aea-44d2-9091-4b3032f7a29a)
